### PR TITLE
test: Adds tests to the TimeSeriesColumnControl component

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Adds tests to the `TimeSeriesColumnControl` component.

### TEST PLAN
1 - Execute `TimeSeriesColumnControl` tests
2 - All tests should pass

@rusackas @junlincc 

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by geido — APPROVED** [review]
Great improvements to the tests! LGTM!

File: superset-frontend/src/explore/components/controls/TimeSeriesColumnControl/TimeSeriesColumnControl.test.tsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  expect(screen.getByText('Type')).toBeInTheDocument();
+});
+
+test('triggers onChange when type changes', () => {
+  const onChange = jest.fn();
+  render(<TimeSeriesColumnControl onChange={onChange} />);
+  userEvent.click(screen.getByRole('button'));
+  userEvent.click(screen.getByText('Select...'));
+  userEvent.click(screen.getByText('Time comparison'));
+  expect(onChange).toHaveBeenCalled();
```
Comment (yardz) [inline superset-frontend/src/explore/components/controls/TimeSeriesColumnControl/TimeSeriesColumnControl.test.tsx:45]: Same thing as the other tests. Any of the previous actions (any one of them) may have triggered the "onChange"

Comment (yardz) [reply superset-frontend/src/explore/components/controls/TimeSeriesColumnControl/TimeSeriesColumnControl.test.tsx:45]: The same observation for almost all tests

Comment (michael-s-molina) [reply superset-frontend/src/explore/components/controls/TimeSeriesColumnControl/TimeSeriesColumnControl.test.tsx:45]: Done.

**Review by rusackas — APPROVED** [review]
LGTM

**Comment by junlincc** [comment]
Screen Shot 2021-04-05 at 11 04 04 AM
lgtm 🟢

**Pull request merged**